Sources/KlaviyoSwift/Klaviyo.swift

```diff
@@ -62,7 +59,10 @@ public struct KlaviyoSDK {
     /// - Returns: a KlaviyoSDK instance
     @discardableResult
     public func initialize(with apiKey: String) -> KlaviyoSDK {
-        dispatchOnMainThread(action: .initialize(apiKey))
+        Task {
+            let appContextInfo = await environment.appContextInfo()
```

Comment:
Retreiving app context requires mainactor access so this can no longer be done inside the reducer. It is now done here and passed through.